core/primitives/src/congestion_info.rs

```diff
@@ -532,203 +444,231 @@ mod tests {
     /// Default congestion info should be no congestion => maximally permissive.
     #[test]
     fn test_default_congestion() {
-        let inner_congestion_info = CongestionInfoV1::default();
+        let config = CongestionControlConfig::default();
+        let info = CongestionInfo::default();
+        let congestion_control = CongestionControl::new(config, info, 0);
 
-        assert_eq!(0.0, inner_congestion_info.memory_congestion());
-        assert_eq!(0.0, inner_congestion_info.incoming_congestion());
-        assert_eq!(0.0, inner_congestion_info.outgoing_congestion());
-        assert_eq!(0.0, inner_congestion_info.congestion_level(0));
+        assert_eq!(0.0, congestion_control.memory_congestion());
+        assert_eq!(0.0, congestion_control.incoming_congestion());
+        assert_eq!(0.0, congestion_control.outgoing_congestion());
+        assert_eq!(0.0, congestion_control.congestion_level(true));
```

Comment:
maybe it would make sense to test with `.congestion_level(true)` and `.congestion_level(false)`.